draft-irtf-cfrg-frost.md

```diff
@@ -1031,12 +1050,316 @@ signature_share
+S2 group_commitment_share: 7e91f66097b6450c52c89c14400a506ee1d37f5e52
+a8d4c3fc9733c23d0b27cd6cfce55a8aee692262e5815be341e8d0b9d240a9630c9f0
+600
+
+sig: 4d9883057726b029d042418600abe88ad3fec06d6a48dca289482e9d51c10353
+37e4d1aae5fd1c73a55701133238602f423886fc134a3c65800a0ed81f9ed29fcafe1
+ee753abef0df8a9686a3fcc0caaca7bbcecd597069f2a74da3f0d97a98e9740e35025
+716ab554d524742c4d0bd83800
+~~~
+
+## FROST(ristretto255, SHA512)
+
+~~~
+// Configuration information
+MAX_SIGNERS: 3
+THRESHOLD_LIMIT: 2
+NUM_SIGNERS: 2
+
+// Group input parameters
+group_secret_key: b120be204b5e758960458ca9c4675b56b12a8faff2be9c94891
+d5e1cd75c880e
+group_public_key: 563b80013f337deaa2a282af7b281bd70d2f501928a89c1aa48
+b379a5ac4202b
+message: 74657374
+
+// Signer input parameters
+S1 signer_share: 94ae65bb90030a89507fa00fff08dfed841cf996de5a0c574f1f
+4693ddcb6705
+S2 signer_share: 641003b3f00bb1e01656ac1818a4419a580e637ecaf67b191521
+2e0ae43a470c
+S3 signer_share: 479eaa4d36b145e00690c07e5245c5312c00cd65b692ebdbda22
+1681eaa92603
+
+// Round one parameters
+participants: 1,2
+commitment_list: 0001824e9eddddf02b2a9caf5859825e999d791ca094f65b814a
+8bca6013d9cc312774c7e1271d2939a84a9a867e3a06579b4d25659b427439ccf0d74
+5b43f75b76600028013834ff4d48e7d6b76c2e732bc611f54720ef8933c4ca4de7eaa
+a77ff5cd125e056ecc4f7c4657d3a742354430d768f945db229c335d258e9622ad99f
+3e758f226c1530c93fbfe1a29f34aa2e13da14ace01b6e6412e36d5e01baba2c78e49
+21dc1c0b7143210bb0fc42553c3a9490ba011e30250727c0189372a38632591f
+group_binding_factor: f49fbf1a092173b9338394b5818966480c0413c5f90e2a7
+65aabc1a10cfb3908
+
+// Signer round one outputs
+S1 hiding_nonce: 349b3bb8464a1d87f7d6b56f4559a3f9a6335261a3266089a9b1
+2d9d6f6ce209
+S1 binding_nonce: ce7406016a854be4291f03e7d24fe30e77994c3465de031515a
+4c116f22ca901
+S1 hiding_nonce_commitment: 824e9eddddf02b2a9caf5859825e999d791ca094f
+65b814a8bca6013d9cc3127
+S1 binding_nonce_commitment: 74c7e1271d2939a84a9a867e3a06579b4d25659b
+427439ccf0d745b43f75b766
+S2 hiding_nonce: 4d66d319f20a728ec3d491cbf260cc6be687bd87cc2b5fdb4d5f
+528f65fd650d
+S2 binding_nonce: 278b9b1e04632e6af3f1a3c144d07922ffcf5efd3a341b47abc
+19c43f48ce306
+S2 hiding_nonce_commitment: 8013834ff4d48e7d6b76c2e732bc611f54720ef89
+33c4ca4de7eaaa77ff5cd12
+S2 binding_nonce_commitment: 5e056ecc4f7c4657d3a742354430d768f945db22
+9c335d258e9622ad99f3e758
+
+// Round two parameters
+participants: 1,2
+
+// Signer round two outputs
+S1 sig_share: 503c943f6af38f3ad2271de3d792c4756c5d9c483989e2005add0e8
+9b504dc01
+S1 group_commitment_share: 40587ff15e9cba4fe601c4378b40ae4babe2ea6f0b
+549a6712d84837166b954d
+S2 sig_share: eee969c17354ec84933723b45ac9965d2874e21c3a50cd31648f708
+bfb9f160b
+S2 group_commitment_share: 70661957faf398410a9da3e20cfd6f1233b368548e
+44ffccee3143a42b691844
+
+sig: 90ac875023a311624948b2660bc5524f690ae9e14fcd6541959bee2e868d8c32
+3e26fe00de477cbf655f4097325c5bd394d17e6573d9af32be6c7f14b1a4f20c
+~~~
+
+## FROST(P-256, SHA256)
+
+~~~
+// Configuration information
+MAX_SIGNERS: 3
```

Comment:
Should we have a test for a different number of signers, just in case?

Reply:
Do you mean a different number for `MAX_SIGNERS`, or `NUM_SIGNERS`? I don't think changing `MAX_SIGNERS` increases test coverage. Changing `NUM_SIGNERS` might, however. I would prefer to do that in a separate PR.